util/logger.js

```diff
@@ -95,10 +85,7 @@ function log(name, data, category) {
  */
 function createDirectory(category) {
 	// Choose the directory
-	let dir = "./log/" + category + "/";
-	if (process.env.CI) { // (Change to a test directory if being ran by mocha.)
-		dir = "./test/log/" + category + "/";
-	}
+	const dir = `./${process.env.CI ? "test/" : ""}${category}`;
```

Comment:
```suggestion
	const dir = `./log/${process.env.CI ? "test/" : ""}${category}`;
```
Fix log files being written to `./` instead of `./log/`